src/main/java/usecases/friendinteractors/viewfriend/view_friend_interactor.java

```diff
@@ -22,6 +22,12 @@ public void ViewFriend(view_friend_input_model model) {
         ArrayList<CommonUser> arraylist = new ArrayList<CommonUser> ();
         arraylist = dataBase.getFriendList(model.requesterid);

```

Comment:
You can shorten this to:
 ArrayList<CommonUser> arraylist = dataBase.getFriendList(model.requesterid);

Also, it should be arrayList.

Reply:
Changed :)